docs/src/main/sphinx/release/release-470.md

```diff
@@ -0,0 +1,105 @@
+# Release 470 (5 Feb 2025)
+
+## General
+
+* Add [](/connector/duckdb). ({issue}`18031`)
+* Add [](/connector/loki). ({issue}`23053`)
+* Add support for `WITH SESSION` to set session properties with `SELECT`
+  queries. ({issue}`24889`)
+* Add support for access control systems to control privileges for
+  `CREATE FUNCTION` and `DROP FUNCTION`. ({issue}`24696`)
+* Improve compatibility of fault-tolerant exchange storage with S3-compliant
+  object stores. ({issue}`24822`)
+* Improve compatibility of fault-tolerant exchange storage with
+  Hadoop-compatible file systems and the HDFS as exchange type by adding the
+  `exchange.hdfs.skip-directory-scheme-validation` property. ({issue}`24627`)
+* Export JMX metric for `blockedQueries`. ({issue}`24907`)
+* {{breaking}} Remove the deprecated precomputed hash optimizer and the related
```

Comment:
"Remove support for the `optimize_hash_generation` session property and `optimizer.optimize-hash-generation` configuration option"

Reply:
done